packages/react-core/src/components/LabelGroup/examples/LabelGroupEditableAddModal.tsx

```diff
@@ -81,7 +86,7 @@ export const LabelGroupEditableAddModal: React.FunctionComponent = () => {
           isEditable: isEditable !== undefined ? isEditable : true,
           ...(isEditable && {
             editableProps: {
-              'aria-label': 'label editable text'
+              'aria-label': `Editable label with text ${labelText || 'New Label'}`
```

Comment:
Similar to the above comment, the "Label 3" label just needs its aria-label updated on line 50.